packages/caip/src/adapters/onramper/index.ts

```diff
@@ -22,16 +35,28 @@ export const AssetIdToOnRamperIdMap: Record<AssetId, OnRamperTokenId[]> = {
   'eip155:1/erc20:0x4e15361fd6b4bb609fa63c81a2be19d873717870': ['FTM'],
   'eip155:1/erc20:0xf57e7e7c23978c3caec3c3548e3d615c346e79ff': ['IMX'],
   'eip155:1/erc20:0x514910771af9ca656af840dff83e8264ecf986ca': ['LINK'],
+  'eip155:1/erc20:0x5f98805a4e8be255a32880fdec7f6728c6568ba0': ['LUSD_ERC20'],
   'eip155:1/erc20:0x9f8f72aa9304c8b593d555f12ef6589cc3a579a2': ['MKR', 'MKR_ERC20'],
   'eip155:1/erc20:0x0f5d2fb29fb7d3cfee444a200298f468908cc942': ['MANA', 'MANA_ERC20'],
+  'eip155:1/erc20:0x69af81e73a73b40adf4f3d4223cd9b1ece623074': ['MASK', 'MASK_ERC20'],
+  'eip155:1/erc20:0xb26631c6dda06ad89b93c71400d25692de89c068': ['MINDS', 'MINDS_ERC20'],
   'eip155:1/erc20:0xd26114cd6ee289accf82350c8d8487fedb8a0c07': ['OMG'],
+  'eip155:1/erc20:0x75231f58b43240c9718dd58b4967c5114342a86c': ['OKB_ERC20'],
+  'eip155:1/erc20:0x83e6f1e41cdd28eaceb20cb649155049fac3d5aa': ['POLS_ERC20'],
   'eip155:1/erc20:0x57ab1ec28d129707052df4df418d58a2d46d5f51': ['SUSDC'],
   'eip155:1/erc20:0x6b3595068778dd592e39a122f4f5a5cf09c90fe2': ['SUSHI', 'SUSHI_ERC20'],
   'eip155:1/erc20:0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f': ['SNX'],
+  'eip155:1/erc20:0xae7ab96520de3a18e5e111b5eaab095312d7fe84': ['STETH'],
+  'eip155:1/erc20:0x0000000000085d4780b73119b644ae5ecd22b376': ['TUSD'],
+  'eip155:1/erc20:0x04fa0d235c4abf4bcf4787af4cf447de572ef828': ['UMA'],
   'eip155:1/erc20:0x1f9840a85d5af5bf1d1762f925bdaddc4201f984': ['UNI', 'UNI-ERC20'],
   'eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48': ['USDC'],
   'eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7': ['USDT'],
   'eip155:1/erc20:0x2260fac5e5542a773aa44fbcfedf7c193bc2c599': ['WBTC'],
+  'eip155:1/erc20:0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2': ['WETH'],
+  'eip155:1/erc20:0x0bc529c00c6401aef6d220be8c6ea1667f6ad93e': ['YFI'],
+  'eip155:1/erc20:0xe41d2489571d322189246dafa5ebde1f4699f498': ['ZRX'],
+  'eip155:43114/erc20:0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e': ['USDC_CCHAIN'],
```

Comment:
Confirmed we good https://snowtrace.io/address/0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e